test/comparison-tests/create-and-execute-test.js

```diff
@@ -402,9 +402,9 @@ function getNormalisedFileContent(file, location) {
                 .replace(/Module build failed \(from \//gm, 'Module build failed (from ')
                 .replace(/Module Warning \(from \//gm, 'Module Warning (from ')
                 // We don't want a difference in the number of kilobytes to fail the build
-                .replace(/[\d]+([.][\d]*)? KiB/g, 'A-NUMBER-OF KiB')
+                .replace(/\s+[\d]+([.][\d]*)? KiB\s+/g, ' A-NUMBER-OF KiB ')
```

Comment:
we're getting some meaningless failures here related to whitespace in webpack output.  These regex changes effectively normalize those differences.  We want meaninful failures only 😄